documentation/en/default-lotus-config.toml

```diff
@@ -40,7 +40,7 @@
   # env var: LOTUS_LIBP2P_LISTENADDRESSES
   #ListenAddresses = ["/ip4/0.0.0.0/tcp/0", "/ip6/::/tcp/0", "/ip4/0.0.0.0/udp/0/quic-v1", "/ip6/::/udp/0/quic-v1", "/ip4/0.0.0.0/udp/0/quic-v1/webtransport", "/ip6/::/udp/0/quic-v1/webtransport"]
 
-  # Addresses to explicitally announce to other peers. If not specified,
```

Comment:
this comes from `node/config/types.go` so you'll need to edit it there and `make gen` or just edit it manually wherever it appears (`git grep explicitally` will show you the 3 places).